### DISCUSSION TITLE
Fan mode change NZXT RGB & Fan Controller - AC-CRFR0-B1-6

### DISCUSSION THREAD

**Discussion by DcryptedCA** [liquidctl/liquidctl#716] [Q&A] [answered]
Hello!

Suddenly my NZXT switched one fan from PWM to DC and I don't know why, but a shutdown/reboot didn't fix it and I tried running variations of the below without success:
liquidctl -m nzxt initialize --fan-mode='1:pwm' -g --direct-access

Any ideas how to get it to change back?

# liquidctl status
NZXT RGB & Fan Controller (3+6 channels) (experimental)
├── Fan 1 control mode      DC  
├── Fan 1 duty              80  %
├── Fan 1 speed            405  rpm
├── Fan 2 control mode     PWM  
├── Fan 2 duty              80  %
├── Fan 2 speed           3592  rpm
├── Fan 3 control mode     PWM  
├── Fan 3 duty              80  %
└── Fan 3 speed           3508  rpm

**Comment by amezin** [comment, marked as answer]
Usually, reinitialization means re-detection of fans for these devices.

You can also try to do a "full reset" by unplugging the controller temporarily from both usb and sata (while the PC is off), or just switching off the power supply for a few seconds.

But most likely you'll see the issue again - I've seen some fans that didn't detect as PWM reliably, one time they were "PWM", other time "DC". When they are plugged into the motherboard, you can force PWM mode in bios, but for this controller there's likely no way to do it.

**Comment by DcryptedCA** [comment]
The fans are all the same, and it was PWM until two days ago (after a clean power outage shutdown)... I shutdown but didn't pull the plug on the server so it still has "power". Its not easily accessible so "remote" control would be preferable (would take me like an hour to get to the NZXT).

**Comment by DcryptedCA** [comment]
The fix here was a shutdown and then disconnect the power supply. Doesn't seem possible from a remote perspective unless I put a remote device on the plug....
